public/main.js

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable no-undef */
```

Comment:
This directive is meant to protect you. If you have some global variable, you can declare it.
```js
/* global flatpickr */
```